### PR TITLE
Fix up snippets round 3

### PR DESCRIPTION
**Pull request by rosieyohannan**
# Description
Fix up some snippet formatting for config.yml examples

# Reasons
copypasteability

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/concepts.adoc [hunk 1]
```diff
@@ -305,34 +356,46 @@ The *machine executor* spins up a complete Ubuntu virtual machine image, giving
...
+          POSTGRES_USER: user
+    steps:
+      - checkout
+#...
+  build2:
+    machine: # Specifies a machine image that uses
+      # an Ubuntu version 20.04 image with Docker 20.10.12
+      # and docker-compose 1.29.2, follow CircleCI Discuss Announcements
+      # for new image releases.
+      image: ubuntu-2004:202201-02
```
Comment (imlogang) [inline jekyll/_cci2/concepts.adoc:381]: This machine image is currently being deprecated/EOLd. 
Would it be worth specifying a newer image? 

We have until Sept 2024, but if we don't want to go back and change it.

Comment (rosieyohannan) [reply jekyll/_cci2/concepts.adoc:381]: Oh yes thanks for pointing this out. will update!

File: jekyll/_cci2/concepts.adoc [hunk 2]
```diff
@@ -118,18 +119,31 @@ jobs:
             - echo-output
 
   build2:
-#...
+    machine:
+      image: ubuntu-2204:2022.07.1
```
Comment (rosieyohannan) [inline jekyll/_cci2/concepts.adoc:123]:
```suggestion
      image: ubuntu-2204:2024.01.2
```

File: jekyll/_cci2/concepts.adoc (hunk 1 above)
Comment (rosieyohannan) [inline jekyll/_cci2/concepts.adoc:381]:
```suggestion
    machine: # Specifies a machine image that uses
      # an Ubuntu version 22.04 image
      image: ubuntu-2204:2024.01.2
```

**Review by imlogang — APPROVED** [review]
LGTM. It's failing the linter but i think it's failing on different pages than what you've updated here.

**Pull request merged**